qiskit/result/distributions/quasi.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, data, shots=None, stddev_upper_bound=None):
         self.shots = shots
         self._stddev_upper_bound = stddev_upper_bound
         self._num_bits = 0
+        self.decimal = 15
```

Comment:
It might be good to set the decimal with a class variable `__decimal__` (instead of instance variable `self.decimal`) similar to `Pauli.__repr__`.
https://github.com/Qiskit/qiskit/blob/55d4e98c864cca9c2e6a577cd941d1e7ceec54bd/qiskit/quantum_info/operators/symplectic/pauli.py#L96-L101
Do you have any suggestion, @ikkoham?

Reply:
I agree with @t-imamichi. Class variable is better than instance variable. Variable names can be something like dunder/magic (`__decimal__`), or they can be normal (`decimal`).

Reply:
Great idea! done in d2866a333060e4b7555b2790b68e8eea065e1a1e

Thanks!